pkg/kubectl/describe.go

```diff
@@ -1801,7 +1833,7 @@ func describeServiceAccount(serviceAccount *api.ServiceAccount, tokens []api.Sec
 			tokenSecretNames = []string{}
 		)
 
-		for _, s := range serviceAccount.ImagePullSecrets {
+		for _, s := range saPullSecrets {
```

Comment:
if you fail to list the secrets at all, I wouldn't add anything to the description. if you successfully list them and some don't exist, I would indicate that in the description